composer.json

```diff
@@ -27,7 +27,7 @@
         "ergebnis/php-cs-fixer-config": "^6.34",
         "friendsofphp/php-cs-fixer": "^3.61",
         "phpstan/phpstan": "^1.11",
-        "phpunit/phpunit": "*",
```

Comment:
I actually wanted to keep it as `"phpunit/phpunit": "*"` so that depending on PHP version used the latest version of PHPUnit is loaded. The library supported many PHP versions and a single PHPUnit version was not able to accommodate it. Well, maybe not a good idea to support many PHP versions at once.

Reply:
I would specify the exact allowed/supported versions, which are 10 and 11 right now.

Reply:
But not sure, why we should not pin 11, as our supported php versions are all working fine with php-unit 11

Reply:
Well, it is either `11` or `*`, no need for `10` as it will never be used anyways. By using `*` we always make sure that latest PHPUnit version is used and when version 12 comes out, tests will use it automatically, depending if this version will support PHP version that this library supports. It may happen that 8.2 tests will run on PHPUnit 11 and 8.3 on PHPUnit 12, when its out. If we pin 11, both tests will use 11. Basically I want to make sure to always use latest PHPUnit version possible.

Reply:
Ok what about >=11.0

Same as we do for PHP

Reply:
BTW open it up could mean if we don't change the code anymore someone will be able to install phpunit 13 even if it breaks something, that's a bad DX IMHO.

We don't need to allow every version as we are not testing phpunit but our code, so saying you need to use version 11 seems appropriate to me. Highest and low deps are only meant for require and not for require dev 

Just my thoughts

Reply:
Yeah, this would work :)

Reply:
#Did you check my last comment also? 😄

Reply:
We also don't do this for phpstan, rector or cs fixer

Reply:
hm, your comment about opening it up makes real sense... lets cap it at 11 for now so.

Reply:
> #Did you check my last comment also? 😄

It have seen it after I replied :D